app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_confirmation.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.gform.config.FrontendAppConfig
+@import uk.gov.hmrc.gform.testonly.{SnapshotId}
+@import uk.gov.hmrc.gform.views.html.govuk_wrapper
+
+@(
+  frontendAppConfig: FrontendAppConfig,
+  snapshotId: SnapshotId,
+  actionUrl: String,
+  backUrl: String
+)(implicit
+  request: Request[_],
+  messages: Messages,
+  l:LangADT
+)
+
+@backLink = @{
+  new BackLink(href = backUrl, content = new Text(messages("linkText.back")))
+}
+
+@{
+    val customContent = Html(s"""
```

Comment:
What is this, why don't you use html directly?

Reply:
but I need to supply it to govuk_wrapper as argument. I create a variable with html.
```    govuk_wrapper(
 ...
      content = customContent,
  ...
    )
```

Reply:
Just do
```
@customContent = {
     <style>...
}
```
And replace all `$` with `@` and you almost good to go.

Reply:
ok, I updated a PR